builder/README.md

```diff
@@ -140,6 +144,8 @@ Note:
 * value of _lockMode only supports `share` and `exclusive` temporarily:
     * `share` representative `SELECT ... LOCK IN SHARE MODE`. Unfortunately, the current version does not support `SELECT ... FOR SHARE`, It'll be supported in the future.
     * `exclusive` representative `SELECT ... FOR UPDATE`
+* key start with `_custom_` the value must impl `builder.Comparable`,eg `Custom`,`JsonContains`;you can also impl `builder.Comparable` yourself
```

Comment:
if key starts with `_custom_`, the corresponding value must be a `builder.Comparable. We provide builtin type such as `Custom` and `JsonContains`. You can also provide your own implementation if you want